tool/jt.rb

```diff
@@ -517,6 +518,26 @@ def dis(file)
     puts Pathname(file).sub_ext('.ll')
   end
 
+  def environment
+    puts "Environment"
+    env_vars = ['JAVA_HOME','RUBY_BIN','GRAALVM_BIN','GRAAL_HOME',
+    'JVMCI_BIN','JVMCI_GRAAL_HOME','GRAAL_JS_JAR',
+    'SL_JAR','OPENSSL_PREFIX','AOT_BIN','TRUFFLERUBY_CEXT_ENABLED',
+    'TRUFFLERUBYOPT','RUBYOPT']
+    env_vars.each do |e|
+      puts "#{e}: #{ENV[e]}"
```

Comment:
`#{ENV[e].inspect}` so we can differentiate empty string and unset variable.

Reply:
Fixed.